src/go/util/httppattern/uri_template.go

```diff
@@ -62,7 +62,7 @@ type variable struct {
 	HasDoubleWildCard bool
 }
 
-func (u *UriTemplate) String() string {
+func (u *UriTemplate) ExactMatchString(acceptTrailingBackslash bool) string {
 	if len(u.Segments) == 0 {
 		return "/"
```

Comment:
Please add unit test of calling this method  on the uriTemplate "/" with true/false, which is the only one we accept with trailing slash/

Reply:
Done